test/controllers/tracks_controller_test.rb

```diff
@@ -282,6 +282,7 @@ class TracksControllerAudioTest < ActionDispatch::IntegrationTest
     location = Location.create(path: Rails.root.join('test/files'))
     flac = Codec.create(mimetype: 'audio/flac', extension: 'flac')
     audio_file = create(:audio_file, location:, filename: '/base.flac', codec: flac)
+    perform_enqueued_jobs
```

Comment:
Most tests will have the jobs executes inline, but Rails sometimes overrides the queue_adapter. We could work around this, but we can just as well force the jobs to execute.
See: https://github.com/rails/rails/issues/37270